.gitignore

```diff
@@ -4,3 +4,4 @@
 *.jl.mem
 /docs/build/
 /docs/site/
+Manifest.toml
```

Comment:
Perhaps add `docs/Manifest.toml` as well?

Reply:
This rule ignores all files named `Manifest.toml`. `/Manifest.toml` would be how to ignore only the root file.

Reply:
Oh whoa, TIL!